Library/Homebrew/cmd/analytics.rb

```diff
@@ -0,0 +1,44 @@
+#:  * `analytics` [`state`]:
+#:    Display analytics state.
+#:
+#:  * `analytics` `on`|`off`:
+#:    Turn on/off Homebrew's analytics.
+#:
+#:  * `analytics` `regenerate-uuid`:
+#:    Regenerate UUID used in Homebrew's analytics.
+
+module Homebrew
+  def analytics
+    config_file = HOMEBREW_REPOSITORY/".git/config"
+
+    case ARGV.named.first
+    when nil, "state"
```

Comment:
I keep it is for now. Although I think it's better to put them into separate methods, I think we should wait when we implement command in class. Otherwise, it's too easy to pollute global namespace.